tests/integration/test_balancer.py

```diff
@@ -29,35 +30,31 @@
 PRODUCER_APP = "producer"
 
 
-@pytest.fixture(params=[APP_NAME, BALANCER_APP], scope="module")
+@pytest.fixture(params=[APP_NAME], scope="module")
```

Comment:
**question:** This would effectively remove the second iteration of the tests, where we have 2 applications rather than 1. Is that intentional?

Reply:
This is a mistake, I should not have committed this line. But since the test does not run, and I will work on it on the next ticket I figured it would be ok to leave it there instead of pushing (and waiting for green CI).

Since I have other things to address, I'll bring it back

Reply:
Brought this param back in c2b646fa2daf